src/SOS/Strike/strike.cpp

```diff
@@ -6644,7 +6644,7 @@ HRESULT PrintThreadsFromThreadStore(BOOL bMiniDump, BOOL bPrintLiveThreadsOnly)
         table.WriteColumn(3, Pointer(CurThread));
         table.WriteColumn(4, ThreadState(Thread.state));
         table.WriteColumn(5,  Thread.preemptiveGCDisabled == 1 ? "Cooperative" : "Preemptive");
-        table.WriteColumnFormat(6, "%p:%p", Thread.allocContextPtr, Thread.allocContextLimit);
+        table.WriteColumnFormat(6, "%p:%p", SOS_PTR(Thread.allocContextPtr), SOS_PTR(Thread.allocContextLimit));
```

Comment:
Without this fix, `Thread.allocContextPtr` was padded to 64 bits confusing varargs.